rotkehlchen/tests/unit/accounting/evm/test_paraswap.py

```diff
@@ -26,10 +36,36 @@
 @pytest.mark.parametrize('accounting_initialize_parameters', [True])
 def test_paraswap_swap_with_fee(accountant: 'Accountant'):
     """Test that the fee in paraswap is handled correctly during accounting"""
-    tx_hash, user_address, contract_address, swap_amount_str, fee_amount_str, receive_amount_str = make_evm_tx_hash(), make_evm_address(), make_evm_address(), '138.2851', '11.308417', '1190.359719'  # noqa: E501
+    tx_hash, user_address, contract_address, aquired_from_address, swap_amount_str, fee_amount_str, receive_amount_str = make_evm_tx_hash(), make_evm_address(), make_evm_address(), make_evm_address(), '138.2851', '11.308417', '1190.359719'  # noqa: E501
```

Comment:
Also the numbers are hard. For accounting tests numbers to use should be simple and easy to calculate by hand.

```suggestion
    tx_hash, user_address, contract_address, acquired_from_address, swap_amount_str, fee_amount_str, receive_amount_str = make_evm_tx_hash(), make_evm_address(), make_evm_address(), make_evm_address(), '138.2851', '11.308417', '1190.359719'  # noqa: E501
```